src/OrchardCore.Modules/OrchardCore.ContentFields/Settings/YoutubeFieldSettingsDriver.cs

```diff
@@ -13,8 +13,8 @@ public override IDisplayResult Edit(ContentPartFieldDefinition partFieldDefiniti
             return Initialize<YoutubeFieldSettings>("YoutubeFieldSetting_Edit", model =>
              {
                  partFieldDefinition.PopulateSettings(model);
-                 model.Height = model.Height != default(int) ? model.Height : 315;
-                 model.Width = model.Width != default(int) ? model.Width : 560;
+                 model.Height = model.Height != default ? model.Height : 315;
```

Comment:
We can't as they are not nullable types.

Here I don't understand why default(int) is used.

Reply:
I assumed it was nullable because I saw the `default` being used. I normally would use a nullable value if I want to have a default value like this. I guess they used `default` here as you could override the default value of int from 0 to something else.

Either way, ignore my comment for this PR since all you are doing here is formatting.

Reply:
Yes, here we could have used `!= 0`.

Reply:
The entire YouTube field should be revisited. Maybe instead we end up creating VideoField or something more generic. A topic for a different day

Reply:
Yes, it never ends ;)